rabbitmq/resource_shovel_test.go

```diff
@@ -24,6 +24,12 @@ func TestAccShovel(t *testing.T) {
 					"rabbitmq_shovel.shovelTest", &shovelInfo,
 				),
 			},
+			{
+				Config: testAccShovelConfig_update,
+				Check: testAccShovelCheck(
```

Comment:
This `testAccShovelCheck` function does not really test the content of `info`, only that the shovel exists and that it has the right name / vhost based on its ID.

If you want to be sure the update works correctly, you'll have to check the info too in this function.